drivers/nutdrv_qx_voltronic.c

```diff
@@ -267,7 +267,7 @@ static int	voltronic_bypass_volt_max(char *value, const size_t len)
 		if (ivn >= 200)
 			return -1;
 
-		if (protocol == 2 || protocol == 2 || protocol == 10 || protocol == 13 || protocol == 14)
+		if (protocol == 2 || protocol == 3 || protocol == 10 || protocol == 13 || protocol == 14)  // •••Sean••• not sure... a guess from the comment above
```

Comment:
@aquette @zykh @clepple : experts wanted here :)

Reply:
@seanm guess seems good. @zykh  would be the best judge here, as Qx expert

Reply:
At least, the original line was apparently bogus :) So if there was some error introduced here, at least @zykh was pinged and would know where to look :-p